src/main/java/gov/cms/madie/services/PackagingService.java

```diff
@@ -67,7 +68,24 @@ public byte[] createMeasurePackage(Measure measure, String accessToken) {
 
   public byte[] createQRDA(Measure measure, String accessToken) {
     log.info("Creating QRDA for measure [{}]", measure.getId());
-    // to be implemented: calling back end QRDA Service
-    return "test qrda".getBytes();
+
+    List<String> qrdas = qrdaService.generateQrda((QdmMeasure) measure, accessToken);
+    if (CollectionUtils.isEmpty(qrdas)) {
+      return new byte[0];
+    }
+
+    String zipName = measure.getEcqmTitle() + "-" + measure.getVersion() + "QDM-TestCases";
+
+    log.info("Adding measure package artifacts to the measure package");
```

Comment:
can we add the measure ID to this log statement so we have a little more context if troubleshooting